gui/src/2D/controls/image.ts

```diff
@@ -162,69 +164,97 @@ module BABYLON.GUI {
         /**
          * Gets or sets image source url
          */
-        public set source(value: Nullable<string>) {
-            if (this._source === value) {
-                return;
-            }
+        public set source(value: Nullable<string | AtlasSourceImage>) {
+            if (value) {
+                let src = undefined;
+                if (typeof value === 'string') {
+                    this._sourceLeft = 0;

```

Comment:
You should not remove the current value if source is an url. As an user I could have set them up before setting the source